custom/icds_reports/static/js/icds_dashboard_utils.js

```diff
@@ -1,33 +1,39 @@
-/* global angular hqImport */
+/* global angular hqDefine hqImport */
```

Comment:
Nitpick: `hqDefine` and `hqImport` don't need to go in here, they're in the list of blessed globals: https://github.com/dimagi/commcare-hq/blob/e149006864f5473886e9cdf3a29a5798560c5985/.eslintrc.js#L11

Reply:
ah good to know, thanks